modules/csg/csg_shape.cpp

```diff
@@ -483,6 +483,46 @@ void CSGShape3D::_update_collision_faces() {
 	}
 }
 
+Ref<ArrayMesh> CSGShape3D::bake_static_mesh() {
+	Ref<ArrayMesh> baked_mesh;
+	if (is_root_shape() && root_mesh.is_valid()) {
+		baked_mesh = root_mesh;
+	} else {
+		baked_mesh.instantiate();
```

Comment:
This will return empty mesh. Is it intended? Should it return null and error?

Reply:
changed to return null ref